sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -1304,6 +1305,28 @@ def test_enet_sample_weight_consistency(
     assert_allclose(reg1.coef_, reg2.coef_, rtol=1e-6)
 
 
+@pytest.mark.parametrize("fit_intercept", [True, False])
+@pytest.mark.parametrize("sparse_container", [None] + CSC_CONTAINERS)
+def test_enet_cv_sample_weight(fit_intercept, sparse_container, global_random_seed):
+    rng = np.random.RandomState(global_random_seed)
+    rng = np.random.RandomState(0)
+
+    X, y = make_regression(
+            n_samples=10000, n_features=5, random_state=10
```

Comment:
You need to pass the `rng` object as the `random_state` parameter to make the `global_random_seed` parameter have any impact on this test.